FlagsmithClient/src/main/java/com/flagsmith/entities/Trait.kt

```diff
@@ -6,20 +6,21 @@ import com.google.gson.annotations.SerializedName
 data class Trait (
     val identifier: String? = null,
     @SerializedName(value = "trait_key") val key: String,
-    @SerializedName(value = "trait_value") val traitValue: Any
+    @SerializedName(value = "trait_value") val traitValue: Any,
+    val transient: Boolean = false,
 ) {
 
-    constructor(key: String, value: String)
-            : this(key = key, traitValue = value)
+    constructor(key: String, value: String, transient: Boolean)
+            : this(key = key, traitValue = value, transient = transient)
 
-    constructor(key: String, value: Int)
-            : this(key = key, traitValue = value)
+    constructor(key: String, value: Int, transient: Boolean)
+            : this(key = key, traitValue = value, transient = transient)
 
-    constructor(key: String, value: Double)
-            : this(key = key, traitValue = value)
+    constructor(key: String, value: Double, transient: Boolean)
+            : this(key = key, traitValue = value, transient = transient)
 
-    constructor(key: String, value: Boolean)
-            : this(key = key, traitValue = value)
+    constructor(key: String, value: Boolean, transient: Boolean)
+            : this(key = key, traitValue = value, transient = transient)
```

Comment:
Unless I'm mistaken, this looks like a breaking change? Shouldn't we add overloaded constructors that set `transient` to `False` by default? Maybe that's already done in the class definition above? 🤔

Reply:
Corrected